.github/workflows/release_mac.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
-        target-architecture: ['x86_64', 'arm64']
+        target-architecture: ['x86_64'] # we build universal build => no 'arm64' needed here
```

Comment:
Wait, why do we need x84_64 here for universal build?

Reply:
correct, removed it